### PR TITLE
Support building dovecot with stemmer

### PR DESCRIPTION
**Pull request by dabrahams**
Using stemmer makes lucene searches more useful.

### CODE REVIEW

File: Library/Formula/dovecot.rb [hunk 1]
```diff
@@ -23,6 +24,11 @@ class Dovecot < Formula
     sha256 "82892f876d26008a076973dfddf1cffaf5a0451825fd44e06287e94b89078649"
   end
 
+  resource "stemmer" do
+    url "https://github.com/snowballstem/snowball.git",
+      :revision => "9b58e92c965cd7e3208247ace3cc00d173397f3c"
+  end
```
Comment (mikemcquaid) [inline Library/Formula/dovecot.rb]: Mind asking upstream for a tagged release? Thanks!

**Comment by mikemcquaid** [comment]
There's also some trailing space here: see `brew audit`.

File: Library/Formula/dovecot.rb (hunk 1 above)
Comment (dabrahams) [inline Library/Formula/dovecot.rb]: Sorry, I don't know what that means. I merely copied this stanza from another formula in the library.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format is `dovecot: support building with stemmer.`.

**Pull request closed without merging**